tools/lsp/common/rename_component.rs

```diff
@@ -6,38 +6,49 @@ use std::path::Path;
 use crate::{common, util};
 
 use i_slint_compiler::diagnostics::Spanned;
-use i_slint_compiler::parser::{syntax_nodes, SyntaxKind};
+use i_slint_compiler::parser::{syntax_nodes, SyntaxKind, TextSize};
 use lsp_types::Url;
 
 #[cfg(target_arch = "wasm32")]
 use crate::wasm_prelude::*;
 
+fn main_identifier(
+    input: &i_slint_compiler::parser::SyntaxNode,
+) -> Option<i_slint_compiler::parser::SyntaxToken> {
+    input.child_token(SyntaxKind::Identifier)
+}
+
+fn main_identifier_text(input: &i_slint_compiler::parser::SyntaxNode) -> String {
```

Comment:
Looks like `i_slint_compiler::parser::identifier_text`  Only that it normalize _ and -, which i think should be done.

Reply:
Ah! I was almost sure there was something.... I was looking for something like that starting with `parse_` though :-)